runtime/Makefile

```diff
@@ -13,7 +13,15 @@ export/php%.zip: docker-images
 	PHP_VERSION=$$(echo $@ | cut -d'/' -f 2 | cut -d'.' -f 1);\
 	rm -f $@;\
 	cd export ; \
-	docker run --rm --entrypoint= bref/$$PHP_VERSION:latest sh -c "cd /opt && zip -qq -y -r - {*,.[!.]*} > /tmp/file.zip && cat /tmp/file.zip -" > $$PHP_VERSION.zip ;
```

Comment:
zip is not included anymore so like in [brefphp/extra-php-extensions](https://github.com/brefphp/extra-php-extensions) we have to use another way to extract /opt filesystem from docker image.